install/troubleshooting_unused_drives_on_storage_nodes.md

```diff
@@ -0,0 +1,252 @@
+    ncn-s003  /dev/sdb  ssd   PHYF016401TE1P9DGN  1920G  Unknown  N/A    N/A    No
+    ncn-s003  /dev/sdc  ssd   PHYF015500N51P9DGN  1920G  Unknown  N/A    N/A    No
+    ncn-s003  /dev/sdd  ssd   PHYF0165010Z1P9DGN  1920G  Unknown  N/A    N/A    No
+    ncn-s003  /dev/sde  ssd   PHYF016500YR1P9DGN  1920G  Unknown  N/A    N/A    No
+    ncn-s003  /dev/sdf  ssd   PHYF016500X01P9DGN  1920G  Unknown  N/A    N/A    No
+    ncn-s003  /dev/sdi  ssd   PHYF0165011H1P9DGN  1920G  Unknown  N/A    N/A    No
+    ncn-s003  /dev/sdj  ssd   PHYF016500TQ1P9DGN  1920G  Unknown  N/A    N/A    No
+    ```
+
+    If there are devices that show `Available` as `Yes` and they are not being automatically added, then those devices may need to be zapped.
+
+    **IMPORTANT:** Prior to zapping any device, ensure that it is not being used.
+
+1. Check to see if the number of devices is less than the number of listed drives in the output from step 1.
+
+    ```bash
+    ceph orch device ls|grep dev|wc -l
+    ```
+
+    Example output:
+
+    ```text
+    24
+    ```
+
+    If the numbers are equal, but less than the `total_osds` calculated, then the `ceph-mgr` daemon may need to be failed in order to get a fresh inventory.
+
+    1. Fail the `ceph-mgr` daemon.
+
+        ```bash
+        ceph mgr fail $(ceph mgr dump | jq -r .active_name)
+        ```
+
+    1. Wait 5 minutes and then re-check `ceph orch device ls`.
+
+        See if the drives are still showing as `Available`. If so, then proceed to the next step.
+
+1. Compare `lsblk` output on each storage node against the device list from `ceph orch device ls`.
+
+    ```bash
+    lsblk
+    ```
+
+    Example output:
+
+    ```text
+    NAME                                                                                                 MAJ:MIN RM   SIZE RO TYPE   MOUNTPOINT
+    loop0                                                                                                   7:0    0   4.2G  1 loop  / run/    rootfsbase
+    loop1                                                                                                  7:1    0    30G  0 loop
+     └─live-overlay-pool                                                                                  254:8    0   300G  0 dm
+    loop2                                                                                                  7:2    0   300G  0 loop
+     └─live-overlay-pool                                                                                  254:8    0   300G  0 dm
+    sda                                                                                                    8:0    0   1.8T  0 disk
+     └─ceph--0a476f53--8b38--450d--8779--4e587402f8a8-osd--data--b620b7ef--184a--46d7--9a99--771239e7a323 254:7    0   1.8T  0 lvm
+    ```
+
+    If a device has an LVM volume like above, then it may be in use. In that case, do the [Option 2](#option-2) check below to make sure that the drive can be wiped.
```

Comment:
Should we add a statement to proceed to [Wipe and add drives](#wipe-and-add-drives) if option 2 does not need to be followed?

Reply:
Done. Please re-review when you have the opportunity.